config/webpack.config.js

```diff
@@ -136,7 +143,13 @@ const config = {
 		// will be removed automatically.
 		new CleanWebpackPlugin(),
 		// MiniCSSExtractPlugin to extract the CSS thats gets imported into JavaScript.
-		new MiniCSSExtractPlugin({ esModule: false, filename: '[name].css' }),
+		new MiniCSSExtractPlugin({
+			esModule: false,
+			filename: filenames.css,
+			moduleFilename: ({ name }) =>
+				name.includes('block') ? filenames.blockCSS : filenames.css,
```

Comment:
I wonder if this matching needs to be tightened a bit? What if I created a CSS file named "menu-block" that wasn't related to the block editor?

Reply:
I believe that `name` is the name of the entry point so this shouldn't be an issue but we can still make this a tighter check. Thanks for calling it out.